sirepo/package_data/static/js/ml.js

```diff
@@ -1616,6 +1668,41 @@ SIREPO.app.directive('neuralNetLayersForm', function(appState, mlService, panelS
 
             }
 
+            function getLayerLevel() {
+                if ($scope.layerTarget){
+                    return $scope.layerTarget.layers;
+                }
+                return appState.models.neuralNet.layers;
+            }
+
+            function setLevelName(){
+                if ($scope.layerTarget){
+                    $scope.lName = $scope.layerTarget.name;
+                    return;
+                }
+                appState.models.neuralNet.name = "x";
```

Comment:
Single quotes (we may need to discuss moving to double quotes to match the new python style?)